package/astarte-message-hub/Config.in

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2023 SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config BR2_PACKAGE_ASTARTE_MESSAGE_HUB
+            bool "astarte-message-hub"
+            depends on BR2_PACKAGE_HOST_RUSTC_TARGET_ARCH_SUPPORTS
+            depends on BR2_PACKAGE_HOST_PROTOBUF_ARCH_SUPPORTS
+            depends on BR2_FORTIFY_SOURCE_NONE
+            select BR2_PACKAGE_CA_CERTIFICATES
+            select BR2_PACKAGE_HOST_RUSTC
+            select BR2_PACKAGE_HOST_PROTOBUF
+            default y
+            help
+              A central service that runs on (Linux) devices for collecting and delivering messages
+              from N apps using 1 MQTT connection to Astarte.
+
+              https://github.com/astarte-platform/astarte-message-hub
+
+if BR2_PACKAGE_ASTARTE_MESSAGE_HUB
+
+    config BR2_PACKAGE_ASTARTE_MESSAGE_HUB_ASTARTE_REALM
+      string "Astarte realm"
+      default "test"
+      help
+            The realm to be used use in Astarte requests.
+
+    config BR2_PACKAGE_ASTARTE_MESSAGE_HUB_ASTARTE_PAIRING_BASE_URL
+        string "Astarte pairing base URL"
+        default "http://localhost:4003"
+        help
+            The base URL to reach Astarte Pairing API.This should be the URL up to (and excluding)
+            /v1.
+
+    config BR2_PACKAGE_ASTARTE_MESSAGE_HUB_ASTARTE_PAIRING_JWT
+        string "Pairing JWT token"
+        default ""
+        help
+            JWT token to authorize device registration.
+
+            Can be obtained with ./generate-astarte-credentials -t pairing -p yourrealmkey.key
+
+    config BR2_PACKAGE_ASTARTE_MESSAGE_HUB_ASTARTE_DEVICE_ID
+        string "Astarte Device ID"
+        default ""
+        help
+            Astarte identifies each device with a 128 bit Device ID which has to be unique within
+            its Realm
+
```

Comment:
Add also the option `grpc_socket_port` that is required for the astarte-message-hub. 
https://github.com/astarte-platform/astarte-message-hub/blob/b870447f882183574531d90fa558f8e52c8b153c/README.md?plain=1#L43

Reply:
Done